recipes/songdkl/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "songdkl" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/songdkl-{{ version }}.tar.gz
+  sha256: 3123ba2e27aa898833cdf4a20672962369503acd334791f64adec25acb7dd7bb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - songdkl = 'songdkl.__main__:main'
```

Comment:
```suggestion
    - songdkl = songdkl.__main__:main
```

Reply:
I used this recpe as an example: https://github.com/conda-forge/spyder-feedstock/blob/main/recipe/meta.yaml#L17